ui/src/groups/GroupAvatar.tsx

```diff
@@ -42,7 +42,8 @@ export default function GroupAvatar({
   loadImage = true,
 }: GroupAvatarProps) {
   const { hasLoaded, load } = useAvatar(image || '');
-  const showImage = hasLoaded || loadImage;
+  const imageIsColor = isColor(image || '');
+  const showImage = hasLoaded || loadImage || imageIsColor;
```

Comment:
I think a lot of the logic is getting conflated here.

```suggestion
  const imageIsColor = isColor(image || '');
  const showImage = image && !calm.disableRemoteContent && !imageIsColor && (hasLoaded || loadImage);
```

Then below we can do:

```
if (imageIsColor) {
    background = image;
  } else {
    background = dark ? '#333333' : '#E5E5E5';
  }
  return showImage ? (
    <img className={cn('rounded', size, className)} src={image} onLoad={load} />
  ) : (
    <ColorBoxIcon
      className={cn('rounded', size, textSize(size), className)}
      color={background}
      letter={title ? symbols[0] : ''}
    />
  );
```